modules/ROOT/pages/6.5.1-release-notes.adoc

```diff
@@ -133,9 +147,8 @@ As a consequence, the Accessibility Checker would, incorrectly, find and report
 Accessibility Checker 3.2.0 correctly ignores elements with `contenteditable="false"` attributes set.
 
 For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].
-**Accessibility Checker** 3.2.0 includes the following fixes, additions, and improvements.
 
-=== No longer allow attempting to fix a table’s lack of headers if the table has no cells.
+==== No longer allow attempting to fix a table’s lack of headers if the table has no cells.
 //#TINY-9753
 
 Previously, the Accessbility Checker plugin assumed a table had at least one row. As a consequence it would, consequent to attempting to produce a user-visible error regarding a table having no header rows, throw an internal error when it checked a table that had no cells at all.
```

Comment:
Typo - Accessbility->  Accessibility

Reply:
```suggestion
Previously, the Accessibility Checker plugin assumed a table had at least one row. As a consequence it would, consequent to attempting to produce a user-visible error regarding a table having no header rows, throw an internal error when it checked a table that had no cells at all.
```